### DISCUSSION TITLE
Record MOVD source/target during loadup

### DISCUSSION THREAD

**Discussion by rmkaplan** [Interlisp/medley#1174] [Yearnings]
MOVD (or MOVD?) is used to replace simple versions of functions that are defined early in the loadup with new versions that are defined later (with a different name).

That makes it difficult to figure out what code is actually running.

MOVD, MOVD? or maybe a new function (SYSMOVD?) should be defined that records the source and target names on a global list that can be inspected later.

**Comment by masinter** [comment]
before designing and implementing this, it would be good to have some more discussion about how you would use it...  there are probably many different use patterns and just recording a list of movds won't be enough.
working on browser, I rewrote it to just call the nu- or old- browser so that masterscope wouldn't get quite as confused.
